src/ui/organisms/Eip7702Table.tsx

```diff
@@ -0,0 +1,329 @@
+      if (info.row.original.contract === 'NONE') {
+        return <span style={{ opacity: 0.5 }}>N/A</span>;
+      }
+
+      const contract = info.row.original.contract;
+
+      if (contract === 'UNKNOWN') {
+        return <span style={{ opacity: 1 }}>Unknown - Send PR!</span>;
+      }
+
+      return (
+        <ExternalLink
+          url={`https://etherscan.io/address/${contract.address}`}
+          defaultLabel={contract.name}
+        />
+      );
+    },
+  });
+  const walletContractSource = columnHelper.display({
+    id: 'contract-source',
+    header: 'Source',
+    cell: (info: CellContext<TableRow, unknown>): React.ReactNode => {
+      if (info.row.original.contract === 'NONE') {
+        return <span style={{ opacity: 0.5 }}>N/A</span>;
+      }
+
+      const contract = info.row.original.contract;
+
+      if (contract === 'UNKNOWN') {
+        return null;
+      }
+
+      if (!contract.sourceCode.available) {
+        return <span style={{ color: 'var(--color-error)' }}>Unavailable</span>;
+      }
+
+      const sourceRefs = refs(contract.sourceCode);
+      const sourceUrl = isNonEmptyArray(sourceRefs)
+        ? nonEmptyGet(nonEmptyGet(sourceRefs).urls)
+        : `https://etherscan.io/address/${contract.address}#code`;
+      const rawUrl = isLabeledUrl(sourceUrl) ? sourceUrl.url : sourceUrl;
+
+      return <ExternalLink url={rawUrl} defaultLabel='Available' />;
+    },
+  });
+  const walletTransactionBatchingColumn = columnHelper.display({
+    id: 'transaction-batching',
+    header: 'Batching',
+    cell: (info: CellContext<TableRow, unknown>): React.ReactNode => {
+      if (info.row.original.contract === 'NONE') {
+        return <span style={{ opacity: 0.5 }}>N/A</span>;
+      }
+
+      return <span style={{ opacity: 0.5 }}>Coming soon</span>;
+    },
+  });
+  const columns: Array<ColumnDef<TableRow, CellValue>> = [
+    rankColumn,
+    walletNameColumn,
+    walletTypeColumn,
+    walletContractColumn,
+    walletContractSource,
+    walletTransactionBatchingColumn,
+  ];
+
+  // Create table
+  const table = useReactTable({
+    data: softwareWalletData,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+  });
+
+  // Render table body based on active tab
+  const renderTableBody = (): React.ReactNode => {
+    return table.getRowModel().rows.map(row => {
+      return (
+        <tr key={row.id} className={'dark:bg-[#141414] dark:hover:bg-[#1a1a1a]'}>
+          {row.getVisibleCells().map(cell => (
+            <td
+              key={cell.id}
+              className={cx(
+                'px-4 py-2 dark:text-gray-200',
+                cell.column.id === 'rank' ? 'text-center' : '',
+              )}
+            >
+              {flexRender(cell.column.columnDef.cell, cell.getContext())}
+            </td>
+          ))}
+        </tr>
+      );
+    });
+  };
+
+  return (
+    <div className='overflow-x-auto'>
+      <div className='overflow-x-auto'>
```

Comment:
are these 2 divs needed? Can't you use only one? I'm checking from GitHub so i can't look at it live but i have the feeling only one div would be enough here.